opcua_plugin/opcua_opc-plc_test.go

```
@@ -1395,91 +1217,71 @@
-			Expect(len(messageBatch)).To(Equal(1))
 
-			ctx2, cancel2 := context.WithTimeout(context.Background(), 10*time.Second)
-			defer cancel2()
+			// Track message timestamps
+			var timestamps []time.Time
+			var timestampsMutex sync.Mutex
 
-			messageBatch, _, err = input.ReadBatch(ctx2)
-			Expect(err).To(Equal(context.DeadlineExceeded)) // there should be no data change
-			Expect(len(messageBatch)).To(Equal(0))
+			err = builder.AddConsumerFunc(func(c context.Context, m *service.Message) error {
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)